modules/overrides/user_config_overrides.go

```diff
@@ -281,23 +294,24 @@ func (o *userConfigOverridesManager) DeleteLimits(ctx context.Context, userID st
 	return nil
 }
 
-func (o *userConfigOverridesManager) getTenantLimits(userID string) (*UserConfigurableLimits, bool) {
+// getCachedTenantLimits will return the UserConfigurableLimits for a tenant from the local cache
+func (o *userConfigOverridesManager) getCachedTenantLimits(userID string) (*UserConfigurableLimits, bool) {
 	o.mtx.RLock()
 	defer o.mtx.RUnlock()
 
 	tenantLimits, ok := o.tenantLimits[userID]
 	return tenantLimits, ok
 }
 
-func (o *userConfigOverridesManager) getTenantLimitsAll() map[string]*UserConfigurableLimits {
+func (o *userConfigOverridesManager) getAllCachedTenantLimits() map[string]*UserConfigurableLimits {
 	o.mtx.RLock()
 	defer o.mtx.RUnlock()
 
 	return o.tenantLimits
 }
 
 func (o *userConfigOverridesManager) Forwarders(userID string) []string {
-	tenantLimits, ok := o.getTenantLimits(userID)
```

Comment:
this will only return configured overrides when configured overrides are set, and runtime overrides when configured overrides are not set.

I am leaning towards keeping it as is. do we want this behaviour and want to merge both Forwarders?